generation/new_client/templates/new-client/owlbot.py.j2

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Google LLC
```

Comment:
Can we put these template files under `generation/new_client`?

Reply:
I prefer to place template files in a separate directory than scripts files.

Reply:
Can you make a subdirectory within generation/?

Reply:
(The better question is) What do you feel is the problem with the current location?

Reply:
It's used only by scripts in `/generation`. So, I think it makes sense to be organized that way. For someone navigating the code, it will cause less head scratching. They can still be in a separate directory: `generation/templates`.

Reply:
For those who navigate code, the current structure is unnecessarily deep. That makes sense. Let me change it less nested.

Reply:
Moved template files from `generation/new_client/templates/new-client` to `generation/new_client/templates`.

Reply:
I think I actually misread the original directory structure. Did you have it in `generation/templates` already? I thought it was in `templates` parallel to `generation`. In that case, sorry for the confusion.